pennylane/devices/tests/test_compare_default_qubit.py

```diff
@@ -27,7 +27,8 @@
 class TestComparison:
     """Test that a device different to default.qubit gives the same result"""
 
-    def test_hermitian_expectation(self, device, tol):
+    @pytest.mark.benchmarks
```

Comment:
Wondering if we need a marker for this? Could we add the `--enable-benchmarks` command to the pytest call instead to enable these to be benchmarked?

Reply:
When we mark we can run like:
```
python -m pytest pennylane/devices/tests/test_compare_default_qubit.py --device=lightning.qubit --benchmark-enable -m benchmarks
```
and only the bench**marked** tests will run. Without it when we run with the benchmarks enabled **all** tests will run, and the ones with benchmarks will benchmark.

Reply:
Ah, I understand. Yep, this is sensible then

Reply:
I found out that we don't need to mark the tests, the benchmarks have an option that allows us to only run benchmarks. I updated the PR info to include that. I will keep the info updated with the best practices.